sections/main-product.liquid

```diff
@@ -71,12 +71,25 @@
             {%- assign media = host_product.featured_media -%}
             <li class="product__media-item grid__item slider__slide{% if media.media_type != 'image' %} product__media-item--full{% endif %}{% if section.settings.hide_variants and variant_images contains media.src %} product__media-item--variant{% endif %}" data-media-id="{{ section.id }}-{{ media.id }}">
               {% render 'product-thumbnail', media: media, position: 'featured', loop: section.settings.enable_video_looping, modal_id: section.id, xr_button: true %}
+              {%- if section.settings.show_image_label -%}
+              <div class="image_label">
+                <div id="image_type">Stock Photo</div>
+              </div>
+              {%- endif -%}
             </li>
           {%- endif -%}
           {%- for media in product.media -%}
             {%- unless media.id == product.selected_or_first_available_variant.featured_media.id -%}
               <li class="product__media-item grid__item slider__slide{% if media.media_type != 'image' %} product__media-item--full{% endif %}{% if section.settings.hide_variants and variant_images contains media.src %} product__media-item--variant{% endif %}" data-media-id="{{ section.id }}-{{ media.id }}">
                 {% render 'product-thumbnail', media: media, position: forloop.index, loop: section.settings.enable_video_looping, modal_id: section.id, xr_button: true %} 
+                {%- if section.settings.show_image_label and forloop.index == 1 -%}
+                <div class="image_label">{%- if product.metafields.kameratori.listing_facts.last.value.image_source != "database|actual" or product.metafields.kameratori.listing_facts.value.last.image_source == null or product.metafields.kameratori.listing_facts == blank -%}
+                  <div id="image_type">Actual Item Pictured</div>
```

Comment:
Translatable string should be used, e.g `{{ 'accessibility.refresh_page' | t }}`, instead of literals.

Reply:
Switched, finnish translations are now quite nice (or at least better).